docker/scripts/Test-AzureStack.ps1

```diff
@@ -1,75 +1,309 @@
+        -Tier "Free" `
+        -NumberofWorkers 1 `
+        -WorkerSize "Small" | Format-Table -AutoSize | Out-String | Log
+    Log -message "Running Get-AzAppServicePlan..."
+    Get-AzAppServicePlan -ResourceGroupName $resourceGroup -Name $appName
+    Log -message "Running Remove-AzAppServicePlan..."
+    Remove-AzAppServicePlan -ResourceGroupName $resourceGroup -Name $appName -Force
+}
+
+function AzsAzureBridgeAdmin
+{
+    Log -message "Running Get-AzsAzureBridgeActivation..."
+    Get-AzsAzureBridgeActivation -ResourceGroupName $resourceGroup | Format-Table -AutoSize | Out-String | Log
+}
+
+function AzsBackupAdmin
+{
+    Log -message "Running Get-AzsBackup..."
+    Get-AzsBackup | Format-Table -AutoSize | Out-String | Log
+}
+
+function AzsCommerceAdmin
+{
+    $todayDate = Get-Date
+    $startDate = $todayDate.AddMonths(-1)
+    $startDate = "$($startDate.Month)/$($startDate.Day)/$($startDate.Year)"
+    $todayDate = "$($todayDate.Month)/$($todayDate.Day)/$($todayDate.Year)"
+    Log -message "Running Get-AzsSubscriberUsage..."
+    Log -message "Using time-span of ${startDate} to ${todayDate}."
+    Get-AzsSubscriberUsage -ReportedStartTime $startDate `
+        -ReportedEndTime $todayDate | Format-Table -AutoSize | Out-String | Log
+}
+
+function AzsComputeAdmin
+{
+    Log -message "Running Get-AzsComputeQuota..."
+    Get-AzsComputeQuota | Format-Table -AutoSize | Out-String | Log
+}
+
+function AzsFabricAdmin
+{
+    Log -message "Running Get-AzsInfrastructureLocation..."
+    Get-AzsInfrastructureLocation | Format-Table -AutoSize | Out-String | Log
+}
+
+function AzsInfrastructureInsightsAdmin
+{
+    Log -message "Running Get-AzsAlert..."
+    Get-AzsAlert | Format-Table -AutoSize | Out-String | Log
+}
+
+function AzsGalleryAdmin
+{
+    Log -message "Running Get-AzsGalleryItem..."
+    Get-AzsGalleryItem | Format-Table -AutoSize | Out-String | Log
+}
+
+function AzsKeyVaultAdmin
+{
+    Log -message "Running Get-AzsKeyvaultQuota..."
+    Get-AzsKeyvaultQuota | Format-Table -AutoSize | Out-String | Log
+}
+
+function AzsNetworkAdmin
+{
+    Log -message "Running Get-AzsNetworkQuota..."
+    Get-AzsNetworkQuota | Format-Table -AutoSize | Out-String | Log
+}
 
-    Log -param "END: $($MyInvocation.MyCommand)"
+function AzsStorageAdmin
+{
+    Log -message "Running Get-AzsStorageQuota..."
+    Get-AzsStorageQuota | Format-Table -AutoSize | Out-String | Log
 }
 
-function Test-UserModules()
+function AzsSubscription
 {
-  Log -param "START: $($MyInvocation.MyCommand)"
+    Log -message "Running Get-AzsSubscription..."
+    Get-AzsSubscription | Format-Table -AutoSize | Out-String | Log
+}
 
-  Import-Module Az.Resources
-  Get-AzResourceGroup | Format-Table -AutoSize
+function AzsSubscriptionAdmin
+{
+    Log -message "Running Get-AzsUserSubscription..."
+    Get-AzsUserSubscription | Format-Table -AutoSize | Out-String | Log
+}
 
-  Log -param "END: $($MyInvocation.MyCommand)"
+function AzsUpdateAdmin
```

Comment:
Please dont add any more tests. This is a great start, lets get this in before we consider adding more!